L/LLVM/common.jl

```diff
@@ -201,10 +212,18 @@ if [ ! -z "${LLVM_WANT_EH_RTTI}" ]; then
     CMAKE_FLAGS+=(-DLLVM_ENABLE_RTTI=ON)
     CMAKE_FLAGS+=(-DLLVM_ENABLE_EH=ON)
 fi
-
-if [[ "${bb_full_target}" != *sanitize* && ( "${target}" == *linux* || "${target}" == *mingw* ) ]]; then
+# Change this to check if we are building with clang?
+if [[ "${bb_full_target}" != *sanitize* && ( "${target}" == *linux* ) ]]; then
     # https://bugs.llvm.org/show_bug.cgi?id=48221
-    CMAKE_CXX_FLAGS+="-fno-gnu-unique"
+    CMAKE_CXX_FLAGS+=(-fno-gnu-unique)
+fi
+
+# LLVM 16 requires `align_alloc`, make it available for Intel Linux platforms
```

Comment:
This hack should probably be removed now

Reply:
Was this solved by the glibc bump?

Reply:
It should have been, yes